modules/api-svc/src/lib/model/common/Enums.js

```diff
@@ -17,8 +17,11 @@ const SDKStateEnum = {
     WAITING_FOR_ACTION: 'WAITING_FOR_ACTION',
     WAITING_FOR_PARTY_ACCEPTANCE: 'WAITING_FOR_PARTY_ACCEPTANCE',
     WAITING_FOR_AUTH_ACCEPTANCE: 'WAITING_FOR_AUTH_ACCEPTANCE',
-    QUOTE_REQUEST_RECEIVED: 'QUOTE_REQUEST_RECEIVED',
     WAITING_FOR_QUOTE_ACCEPTANCE: 'WAITING_FOR_QUOTE_ACCEPTANCE',
+    QUOTE_REQUEST_RECEIVED: 'QUOTE_REQUEST_RECEIVED',
+    FX_QUOTE_REQUEST_RECEIVED: 'FX_QUOTE_REQUEST_RECEIVED',
+    FX_QUOTE_WAITING_FOR_ACCEPTANCE: 'FX_QUOTE_WAITING_FOR_ACCEPTANCE',
```

Comment:
Are we using this state 'FX_QUOTE_WAITING_FOR_ACCEPTANCE'?

Reply:
here's:
https://github.com/mojaloop/sdk-scheme-adapter/blob/feat/fx-impl-3618/modules/api-svc/src/lib/model/InboundTransfersModel.js#L547